package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "vscode-powerquery",
-    "version": "0.1.40",
+    "version": "0.1.41",
```

Comment:
We're confident this doesn't need its own feature flag to enable/disable while it is under development?

Reply:
I feel confident as malformed tokens show up as informational errors, it doesn't hard crash the extension.